app/routes/articles/index.js

```diff
@@ -24,5 +24,11 @@ export default IndexRouteUnauthenticated.extend(PagedModelRouteMixin, {
         canAccess: this.can.can('create articles')
       }
     ];
-  })
+  }),
+
+  model(params) {
+    params.paramMapping = this.paramMapping;
+    params.sort = '-pinned,-created_at';
```

Comment:
This breaks the sorting bar probably. So I think we should do something like this `params.sort = '-pinned' + params.sort`